src/Views/gagepage.html

```diff
@@ -250,26 +250,78 @@ <h4 class="modal-title">StreamStats Gage Page</h4>
                             <input type="checkbox" ng-if="vm.formattedDischargePeakDates.length > 0" id="logScaleDischarge" name="logScaleDischarge" ng-model="vm.logScaleDischarge" ng-change="vm.toggleLogLinearDischarge()">
                             <label for="logScaleDischarge" ng-if="vm.formattedDischargePeakDates.length > 0"> Log Scale</label>
 
-                            <input type="radio" ng-if="vm.error == false" id="ageData" name="ageQualityData" ng-model="vm.ageQualityData" value="age" ng-change="vm.toggleDischargeData('age')"> 
+                            <input type="radio" ng-if="vm.error == false" id="ageData" name="toggleDischargeData" ng-model="vm.ageQualityData" value="age" ng-change="vm.toggleDischargeData('age')"> 
```

Comment:
I suggest handling this with an updated `updateChart` function.

```suggestion
                            <input type="radio" ng-if="vm.error == false" id="ageData" name="toggleDischargeData" ng-model="vm.ageQualityData" value="age" ng-change="vm.updateChart()"> 
```